### ISSUE TITLE
Remove links to FIX engine docs

### ISSUE THREAD

**Issue by mattjbray** [imandra-ai/imandra-docs#199]
Request from @ignaden 

https://docs.imandra.ai/

<img width="387" alt="image" src="https://github.com/imandra-ai/imandra-docs/assets/81417/0b6abfbc-f379-476b-91e3-b3764dd75f27">
<img width="578" alt="image" src="https://github.com/imandra-ai/imandra-docs/assets/81417/5cbfd2bd-15cd-43ea-aa21-6c49380388f7">

**Comment by actionshrimp** [comment]
This has been implemented for all sites. It's deployed for:
- the [docs homepage](https://docs.imandra.ai)
- the [IPL docs](https://docs.imandra.ai/ipl/)
- the [rule synth docs](https://docs.imandra.ai/imandra-rule-synth/)

as these sites dont have a 'dev' equivalent.

It's also deployed on the [Imandra docs-dev](https://docs.imandra.ai/imandra-docs-dev/) site, and will be put on the main https://docs.imandra.ai/imandra-docs/ when we do a release of `imandra-docs` - I believe that's currently blocked by the intro page being WIP on master @ignaden (as you have a follow-up PR open), but this will go out with the next imandra-docs release.
